easybuild/tools/filetools.py

```diff
@@ -296,7 +408,7 @@ def find_easyconfigs(path, ignore_dirs=None):
                 continue
 
             spec = os.path.join(dirpath, f)
-            _log.debug("Found easyconfig %s" % spec)
+            _log.debug("Found easyconfig %s", spec)
```

Comment:
@JensTimmerman what is so special about the `log.error` in EB? the only thing that doesn't get formatted is the raise message, but the the call the the regular fancylogger error accepts this.
the only thing that might need fixing is the raise message imho. (`msg` -> `msg % tuple(args)` in the error method)